crates/ef-testing/src/evm_sequencer/evm_state/v0.rs

```diff
@@ -365,23 +363,10 @@ mod tests {
         assert!(tx_output.success);
 
         // Then
-        let evm_address: FeltSequencer = (*TEST_CONTRACT_ADDRESS).try_into().unwrap(); // infallible
-        let kakarot_address: Felt = *KAKAROT_ADDRESS.0.key();
-        let contract_starknet_address = compute_starknet_address(
-            &TEST_CONTRACT_ADDRESS,
-            UNINITIALIZED_ACCOUNT_CLASS_HASH.0,
-            &[kakarot_address, evm_address.into()],
-        )
-        .try_into()
-        .unwrap();
-
         let storage = sequencer
-            .state_mut()
-            .get_storage_at(
-                contract_starknet_address,
-                get_storage_var_address(ACCOUNT_STORAGE, &[Felt::ZERO, Felt::ZERO]),
-            )
+            .storage_at(&TEST_CONTRACT_ADDRESS, U256::ZERO)
             .unwrap();
-        assert_eq!(storage, Felt::ONE);
```

Comment:
i dont understand what changed here 🤔

Reply:
All this code was replicating the  behavior of the function `sequencer.storage_at(<ARGS>)`. So now, it actually use it.

Reply:
ok indeed, its a bit confusing to have storage_at and get_storage_at being two entirely different htings